stl/inc/execution

```diff
@@ -1274,13 +1274,13 @@ using _Parallel_find_results = conditional_t<_Use_atomic_iterator<_FwdIt>, _Para
     _Parallel_choose_min_chunk<_FwdIt>>;
 
 template <class _FwdIt, class _Find_fx>
-struct _Static_partitioned_find2 {
+struct _Static_partitioned_find3 {

```

Comment:
I see that structs are being renamed to:

```
_Static_partitioned_find3
_Static_partitioned_find_end_forward2
_Static_partitioned_find_end_backward3
_Static_partitioned_adjacent_find3
_Static_partitioned_mismatch3
_Static_partitioned_search3
_Static_partitioned_search_n3
_Static_partitioned_is_sorted_until2
_Static_partitioned_is_heap_until2
```

However, I see no motivation for doing so. Am I missing something, or is this a relic of an intermediate edit?

Reply:
(see discord https://discordapp.com/channels/737189251069771789/737189251497721887/1004167710277648384)

Reply:
Resolution: `_Use_atomic_iterator` can return different answers now, so anything that depends on it directly or indirectly (e.g. through `_Parallel_find_results`) should be renamed.